### DISCUSSION TITLE
A newbie. How do I create a new page ?

### DISCUSSION THREAD

**Discussion by Gismet** [cotes2020/jekyll-theme-chirpy#1653] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

I have used the Chirpy starter repository. I can't see _layouts. It seems like pages are generated based on the layouts in the _layouts folder. I want to get rid of the links at the footer of the sidebar but I do not know in which layout they reside. 

### Operations you have already tried

I removed the _sites folder from the .gitignore file and then tried to change index.html files for each tap. But that did not work. The next time I built the project again, all the changes I made reverted to the old version.

**Comment by kungfux** [comment, marked as answer]
Check Chirpy guide on how to [Write a New Post](https://chirpy.cotes.page/posts/write-a-new-post/#naming-and-path).

When you use Chirpy starter, you use Chirpy like a library. That's why you don't see `_layouts` folder, however layout definitions are still exist in a library.

You don't need to keep `_site` folder. It is being created during site build when you publish it through GitHub Actions. If you want to see changes you make locally, please refer to [Running Local Server](https://chirpy.cotes.page/posts/getting-started/#running-local-server). Please don't touch anything under `_site` folder. The idea of Jekyll is to avoid working with HTML and rather write posts using markdown. So, you don't need to save and push `_site` folder to repository.
